src/entities/notesubfolder.cpp

```diff
@@ -60,6 +60,7 @@ NoteSubFolder NoteSubFolder::fetch(int id) {
     return NoteSubFolder();
 }
 
+
```

Comment:
Do you need that newline?

Reply:
Not really :D
artifacts from refactoring